recipes/libssh2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "libssh2" %}
+{% set version = "1.6.0" %}
+{% set sha256 = "5480b3d7b01cd554a7d2671e8ab62f6c10407013db31fb8c3869223521ceda65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+      - vc9    # [win and py27]
+      - vc10   # [win and py34]
+      - vc14   # [win and py35]
+
+requirements:
+  build:
+    - cmake >2.8
+    - openssl 1.0.*
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
+    
+  commands:
+    - test -f $PREFIX/lib64/libssh2.a                 # [not win]
+    - if not exist %LIBRARY_LIB%\\libssh2.lib exit 1  # [win]
```

Comment:
Yeah, with CMake that is true. So what I would do is create a `for`-loop around it and build/install once with shared off and once with shared on. Make sure to clean everything between builds just in case.